pkgs/applications/emulators/86box/default.nix

```
@@ -19,10 +20,12 @@
   discord-gamesdk,
   libpcap,
   libslirp,
+  wayland,
 
   enableDynarec ? with stdenv.hostPlatform; isx86 || isAarch,
   enableNewDynarec ? enableDynarec && stdenv.hostPlatform.isAarch,
   enableVncRenderer ? false,
+  enableWayland ? false,
```

```suggestion
  enableWayland ? stdenv.isLinux,
```

The aur package always enables wayland so it should be fine https://aur.archlinux.org/packages/86box?all_deps=1#pkgdeps

Also add `86Box:` to the commit message.